playbooks/installation/installation.adoc

```diff
@@ -285,35 +113,22 @@ Overall requirements for Installing OpenShift are very simple:
 ** rhel-7-server-ose-3.0-rpms
 * If you plan to Install HA Masters, an additional channel is needed on Master Nodes
 ** rhel-7-for-ha-server-rpms
-* Remove all `NetworkManager*` packages
-
-.Update Needed
-****
-Verify this step is still needed in the official documentation
-****
 * Install the following extra packages: `yum install wget git net-tools bind-utils iptables-services bridge-utils`
 * Optional: install the following diagnostic tools: `yum install lsof strace nc telnet`
 * Fully update all packages: `yum -y update`
-* Configure Storage according to <<storage-considerations>>
 * Sync SSH keys from masters to all nodes (HINT: Here's a script to do this: TODO)
 
```

Comment:
could point to ensuring host access [1] for the TODO. Seems like this can be removed since this info is now provided (unless you has something else in mind).

[1] https://docs.openshift.com/enterprise/latest/install_config/install/prerequisites.html#ensuring-host-access